### ISSUE TITLE
503 error page language declaration

### ISSUE THREAD

**Issue by cyberbaloo** [suitenumerique/docs#1975]
## Bug Report

**Problematic behavior**
When you're on the 503 error, an error page is displayed. The declared language is not correct.

**Expected behavior/code**
Declare the language in the original language of the text: if the error page is in French, there must be `lang="fr"` on the `<html>`

**Additional context/Screenshots**
<img width="1114" height="726" alt="Image" src="https://github.com/user-attachments/assets/5c8527b0-42cf-46aa-aaca-8564247b1b36" />

**Critère RGAA**
8.3 [Dans chaque page web, la langue par défaut est-elle présente ?](https://accessibilite.numerique.gouv.fr/methode/criteres-et-tests/#8.3)

**Impact**
The importance of having the language correctly declared on this page is most critical for screen readers to choose the right voice for speech synthesis and its pronunciation rules. (example: a page in French read with an English voice)
Some braille systems also depend on the language declaration; an incorrect choice can display an incorrect braille transcription.

**Comment by AntoLC** [comment]
This page is not managed in Docs, it appears in rare case when the host has a problem I think. 
Developers cannot update this page.

**Comment by virgile-dev** [comment]
@cyberbaloo as Antho said this page is not Docs specific. We could make one that's generic for all the LaSuite tools. @pierreozoux I think you wanted to look into this. Maybe @robin-lecomte can propose a design  for this.

**Comment by pierreozoux** [comment]
A IH, On utilise https://forge.libre.sh/indiehosters/teapot

On le définit comme backend par default du nginx, et ensuite chaque équipe dev peut à tout moment, en ajoutant l'annotation créer une maitnenance plus spécifique.

**Comment by cyberbaloo** [comment]
@pierreozoux The idea was rather to improve the error pages even if you use Teapot.

It's not out of the question to propose page structure improvements for example. And even better if @robin-lecomte  can propose a design specific to the Suite numérique 🙂

**Comment by robin-lecomte** [comment]
Please see issue #1972 for the design.

**Comment by AntoLC** [comment]
I will close it, as we cannot do anything about it from this repository.
